ubtue_Mohr Siebeck.js

```diff
@@ -1,6 +1,6 @@
 {
 	"translatorID": "30f0052d-e8fc-45ac-a1db-7a729f0da376",
```

Comment:
Leaving the translatorID unchanged leads to weird behaviour like deleting the previously found translator in Zotero-Standablone. Please generate a new hash.